libraries/ESP8266WiFi/src/ESP8266WiFiSTA.h

```diff
@@ -48,7 +48,7 @@ class ESP8266WiFiSTAClass: public LwipIntf {
         bool config(IPAddress local_ip, IPAddress gateway, IPAddress subnet, IPAddress dns1 = (uint32_t)0x00000000, IPAddress dns2 = (uint32_t)0x00000000);
 
         bool reconnect();
-        bool disconnect(bool wifioff = false);
+        bool disconnect(bool wifioff = false, bool eraseap = false);

```

Comment:
Thanks !
I'd approve but I remember @devyte saying that in case of an API addition it should be a principle to leave the original declaration as-is and add a new one for the API extension, in order to not break any overload that would exist in other projects using this one.
This would give:
```cpp
  bool disconnect(bool wifioff = false);
+ bool disconnect(bool wifioff, bool eraseap);
```